### ISSUE TITLE
A couple of doubts

### ISSUE THREAD

**Issue by Akshay199456** [uhcs2320/hwFour#9]
1. I see in the output files that there is no newline at the end of the output. 
![image](https://cloud.githubusercontent.com/assets/11953567/20031534/48c896b2-a347-11e6-9414-022f6848aa64.png)

Is that the intended output or should there be a newline as shown below:

![image](https://cloud.githubusercontent.com/assets/11953567/20031542/683bcc44-a347-11e6-89cb-50280d2b98b5.png)

2. Do you want us to write the code for seats less than 9 or can we assume that all the files have at least 9 seats ?

**Comment by aleman** [comment]
1. Every line should have an end of line character, such as output via:
   `cout << bus << "\t" << name << endl;`
2. No. The minimum number of seats in a bus is 9.

**Comment by aleman** [comment]
A few of the output files had the endline missing on the very last line, but I corrected them several days ago.

**Comment by Akshay199456** [comment]
I might be reading this wrong but this is what I see in the commits made:

![image](https://cloud.githubusercontent.com/assets/11953567/20031830/ed716e42-a34b-11e6-84fb-81bfec0401b6.png)

I don't see an additional line 21 which reflects the newline

**Comment by aleman** [comment]
The output files must not have a single last line that is an empty line.

It is not visible in the line 20 you're referring to, but this is what is going on:

The old version of the file was (wrongly) output via something such as 
`cout << bus << "\t" << name;`

The updated version of the file was output via something such as 
`cout << bus << "\t" << name << endl;`

In the old version, when doing a `cat` in linux, you can see that the last line was missing the endline, and the command prompt wraps immediately next to it.
![screen shot 2016-11-05 at 11 52 26 am](https://cloud.githubusercontent.com/assets/4203458/20031950/704f38ec-a34e-11e6-9064-5a47928f1176.png)

The old version looks like this when doing the `cat` command:
![screen shot 2016-11-05 at 11 53 47 am](https://cloud.githubusercontent.com/assets/4203458/20031955/8a5d0944-a34e-11e6-969d-5adb0fe8fe4c.png)

In summary, as long as **all** lines that output anything to the screen are as follows, your program should be fine:
`cout << bus << "\t" << name << endl;`

(assuming `bus` and `name` are variables in your program, likely of type `string` both of them).

Your program should not have an additional endline being output, that is, the following will be unnecessary and wrong: `cout << endl;`
